### PR TITLE
refactor lerna packages

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,17 +21,19 @@
...
     "postinstall": "npm run lerna",
-    "app:desktop": "lerna run start --scope nteract --stream",
+    "//lerna bootstrap": "builds packages",
+    "lerna": "lerna bootstrap",
+    "build:apps": "lerna run build --scope nteract --scope @nteract/play --scope @nteract/commuter --scope @nteract/notebook-on-next --scope @nteract/showcase",
+    "app:desktop": "lerna build --scope nteract --stream",
     "app:play": "lerna run dev --scope @nteract/play --stream",
     "app:showcase": "lerna run dev --scope @nteract/showcase --stream",
-    "app:commuter": "lerna run dev --scope @nteract/commuter --stream",
+    "app:commuter": "lerna build --scope @nteract/commuter --stream",
```
Comment (rgbkrk) [inline package.json]: We still need `app:commuter` to do the `lerna run dev` command, as it should run the app locally for us.

File: applications/showcase/package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "description": "showcase of nteract components",
   "main": "index.js",
   "scripts": {
-    "dev": "next"
+    "dev": "next",
+    "build": "next build",
+    "start": "next start"
```
Comment (rgbkrk) [inline applications/showcase/package.json:9]: 👍

File: package.json [hunk 3]
```diff
@@ -21,6 +21,9 @@
   "homepage": "https://github.com/nteract/nteract",
   "scripts": {
     "postinstall": "npm run lerna",
+    "//lerna bootstrap": "builds packages",
+    "lerna": "lerna bootstrap",
+    "build:apps": "lerna run build --scope nteract --scope @nteract/play --scope @nteract/commuter --scope @nteract/notebook-on-next --scope @nteract/showcase",
```
Comment (mpacer) [inline package.json]: I don't think you need to have --scope repeated each time.

File: package.json [hunk 4]
```diff
@@ -21,6 +21,9 @@
   "homepage": "https://github.com/nteract/nteract",
   "scripts": {
     "postinstall": "npm run lerna",
+    "//lerna bootstrap": "builds packages",
```
Comment (rgbkrk) [inline package.json:24]: The `lerna bootstrap` symlinks and installs all packages.

File: package.json (hunk 3 above)
Comment (rgbkrk) [reply package.json]: Oh really? I didn't realize that.

**Pull request merged**